pkg/kubelet/apis/config/types.go

```diff
@@ -353,6 +353,10 @@ type KubeletConfiguration struct {
 	// The purpose of this format is make sure you have the opportunity to notice if the next release hides additional metrics,
 	// rather than being surprised when they are permanently removed in the release after that.
 	ShowHiddenMetricsForVersion string
+	// LogFormat Flag specifies the structure of log messages.
+	// this flag accept `text` and `json` options
+	// Refer [structured logging](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1602-structured-logging/README.md) doc for more information.
```

Comment:
Let's link to `component-base/options.go` instead of KEP. 
KEPs can be deprecated/superseeded, let's link to implementation of `--logging-format` flag itself